site/content/docs/main/troubleshooting.md

```diff
@@ -168,6 +174,31 @@ Follow the below troubleshooting steps to confirm that Velero is using the corre
     <Output should be your credentials>
     ```
 
+### Troubleshooting `BackupStorageLocation` credentials
+
+Follow the below troubleshooting steps to confirm that Velero is using the correct credentials if using credentials specific to a [`BackupStorageLocation`][10]:
+1. Confirm that the object storage provider plugin being used supports multiple credentials.
+   If the error message contains `"config has invalid keys credentialsFile"`, the version of your object storage plugin does not yet support multiple credentials.
+   Object storage plugins [provided by the Velero team][11] support this feature, so update your plugin if you are using one of these.
+   If you are using a plugin from a different provider, please contact them for further advice.
+
+1. Confirm that the secret and key referenced by the `BackupStorageLocation` exists in the Velero namespace and has the correct content:
+   ```bash
+   # Determine which secret and key the BackupStorageLocation is using
+   BSL_SECRET=$(kubectl get backupstoragelocations.velero.io -n velero <bsl-name> -o yaml -o jsonpath={.spec.credential.name})
+   BSL_SECRET_KEY=$(kubectl get backupstoragelocations.velero.io -n velero <bsl-name> -o yaml -o jsonpath={.spec.credential.key})
+
+   # Confirm that the secret exists
+   kubectl -n velero get secret $BSL_SECRET
+
+   # Print the content of the secret and ensure it is correct
+   kubectl -n velero get secret $BSL_SECRET -ojsonpath={.data.$BSL_SECRET_KEY} | base64 --decode
+   ```
+   If the secret can't be found, the secret does not within the Velero namespace and must be created.
```

Comment:
a small typo, `If the secret can't be found, the secret does not within the Velero` should be `If the secret can't be found, then secret does not exist within the Velero`